### PR TITLE
[ES-244938] Fix pipelines list pagination

### PR DESCRIPTION
**Pull request by dylanstee-db**
The `pagination` request and response property is now deprecated. Instead, the nested properties from inside `pagination` have been elevated to the top level (where `pagination` used to reside). This PR fixes the pagination-related property access so that more than one page of pipelines are returned.

### Before

```bash
$ databricks pipelines list | jq '. | length'              
25
```

### After

```bash
$ databricks pipelines list | jq '. | length'              
311
```

### CODE REVIEW

**Review by mukulmurthy — APPROVED** [review]
LGTM, have you done a manual test with this against dogfood / some real env?

**Comment by dylanstee-db** [comment]
> have you done a manual test with this against dogfood / some real env?

Yep, the commands in the description are run against e2-dogfood.

**Comment by dylanstee-db** [comment]
@mukulmurthy, do you have permission to merge the PR? I don't have write access to the repo so I can't merge this.

**Comment by dylanstee-db** [comment]
Confirmed that the CLI works on Python v2.7

Screen Shot 2022-01-19 at 3 48 47 PM

**Comment by mukulmurthy** [comment]
Looks like the Python 2.7 build is broken (even on master), but the 3.6 one passed. Sync'd with Deka who gave us permission to merge this.

**Pull request merged**
